### DISCUSSION TITLE
Does this work with DNS-based ad blockers?

### DISCUSSION THREAD

**Discussion by RuinTD** [dhowe/AdNauseam#2524] [Q&A]
If I use a DNS-based ad blocker on my network (such as Pi-hole), does AdNauseam still click on ads? If not, is it possible to have AdNauseam use a different DNS (like 1.1.1.1)?

**Comment by mneunomne** [comment]
@RuiNtD I don't believe this is possible. For us to click on the ads in the first place, they need to be rendered (and then hidden), which means that the browser needs to run the scripts that are coming from some of the domains that the Pi-hole would be blocking. I never worked with Pi-hole

Another option is to simply allow the same domains we `adn-allow` on your Pi-hole, the issue is that the pi-hole "whitelist" are network-wide, so you would have to allow all those domains to all devices connected to that Pi-hole setup, and I guess that's not what you are looking for.

**Reply by RuinTD** [reply]
Fwiw, I'm not actually using Pi-hole. I just used it as an example. I'm using Tailscale with [Mullvad's ad-blocking DNS](https://mullvad.net/en/help/dns-over-https-and-dns-over-tls), so I don't even have control over a whitelist.

**Comment by Mikaela** [comment]
You could possibly use DNS over HTTPS in your browser separate of your system DNS
